shelf_multipart/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: shelf_multipart
 description: shelf_multipart parses multipart and form-data requests for shelf handlers.
-version: 2.0.0
+version: 3.0.0
```

Since there are no changes, I think semver should be

```suggestion
version: 2.0.1
```